core/src/main/scala/chisel3/Data.scala

```diff
@@ -103,10 +103,13 @@ object ActualDirection {
 
   case class Bidirectional(dir: BidirectionalDirection) extends ActualDirection
 
+  /** Converts a `SpecifiedDirection` to an `ActualDirection`
+    *
+    * Implements the Chisel convention that Flip is Input and unspecified is Output.
+    */
   def fromSpecified(direction: SpecifiedDirection): ActualDirection = direction match {
-    case SpecifiedDirection.Unspecified | SpecifiedDirection.Flip => ActualDirection.Unspecified
-    case SpecifiedDirection.Output                                => ActualDirection.Output
-    case SpecifiedDirection.Input                                 => ActualDirection.Input
+    case SpecifiedDirection.Output | SpecifiedDirection.Unspecified => ActualDirection.Output
```

Comment:
is this actually sound ...? Output is coercing while Unspecified is not. I dont think this is equivalent

Reply:
It's unstated in the ScalaDoc but this function is only ever used to determine the `ActualDirection` for `Elements`. It wouldn't be correct for any Aggregate because it would need to handle `ActualDirection.Bidirectional(_)` As for coercion, `chisel3.Output` the operator coerces, but the `ActualDirection` it just reporting the final resolved direction for something after all flips and coercions from above have been applied.